xbmc/cores/VideoPlayer/DVDCodecs/Video/DXVA.cpp

```diff
@@ -198,9 +199,10 @@ static std::string GUIDToString(const GUID& guid)
 
 static const dxva2_mode_t *dxva2_find_mode(const GUID *guid)
 {
-    for (unsigned i = 0; dxva2_modes[i].name; i++) {
-        if (IsEqualGUID(*dxva2_modes[i].guid, *guid))
-            return &dxva2_modes[i];
+	for (const dxva2_mode_t& mode : dxva2_modes)
+	{
+        if (IsEqualGUID(*mode.guid, *guid))
+            return &mode;
```

Comment:
cosmetics: indentation is off for the whole block

Reply:
will fix it